client-src/elements/form-definition.ts

```diff
@@ -1,7 +1,119 @@
-import {html} from 'lit';
+import {html, TemplateResult} from 'lit';
 import * as enums from './form-field-enums';
 
-const COMMA_SEPARATED_FIELDS = [
+interface BaseBrowser {
+  view: {
+    val: string;
+    text: string;
+    url: string;
+    notes: string;
+  };
+}
+
+interface ChromeBrowser extends BaseBrowser {
+  bug: string;
+  blink_components: string[];
+  devrel: string[];
+  owners: string[];
+  prefixed: boolean;
+  status: {val: string};
+  desktop: string;
+  android: string;
+  webview: string;
+  ios: string;
+}
+
+interface FirefoxBrowser extends BaseBrowser {}
+interface SafariBrowser extends BaseBrowser {}
+interface WebDevBrowser extends BaseBrowser {}
+interface OtherBrowser {
+  view: {notes: string};
+}
+
+//resources and standards used in Feature object
+interface Resources {
+  docs: string[];
+  samples: string[];
+}
+
+interface Standards {
+  spec: string;
+  maturity: {val: number; text: string};
+}
+
+type FeatureName = string;
+
+interface Feature {
```

Comment:
Eventually, this should be defined next to https://github.com/GoogleChrome/chromium-dashboard/blob/cfc58fc1e83763107f6c2a268db1a8ae568b0c29/client-src/js-src/cs-client.js#L277 or even in https://github.com/GoogleChrome/chromium-dashboard/blob/main/openapi/api.yaml. But I think defining it here is a reasonable intermediate state.

Reply:
you mean just the type or the feature interface as well?

Reply:
The Feature interface.

Reply:
will fix this after migrating all element files